README.md

```diff
@@ -111,6 +111,7 @@ Power Tuner UI repository: [mattymoomoo/aws-power-tuner-ui](https://github.com/m
 
 From most recent to oldest, with major releases in bold:
 
+* *4.3.4* (2024-02-07): optionally include all results in output
```

Comment:
I'll take care of versioning docs once a new version is pushed out (usually including multiple updates) - we can just remove this for now :)

Reply:
removed